### PR TITLE
Translation Pipeline PR

### PR DESCRIPTION
**Pull request by guacbot**
This auto-generated PR contains the latest completed translation files downloaded from Transifex.
 Preview: https://docs-staging.datadoghq.com/guacbot/translation-pipeline

### CODE REVIEW

File: content/ja/security/cloud_workload_security/workload_security_rules.md [hunk 1]
```diff
@@ -81,6 +81,9 @@ Agent 式は、ホストやコンテナ内のアクティビティに基づい
 例えば、1 つのルールを除いて、open イベントの評価に明示的なファイル名 (例えば `open.file.path == "/etc/shadow”`) を使用し、その 1 つのイベントでワイルドカード (例えば `open.file.path == "/etc/*”`) を使用すると、open イベントは承認者を生成しませんが、実行中に廃棄者を生成するかもしれません。
 
 一般的に、承認者はより強力で好まれます。承認者を使うことで、Agent は何をフィルターにかけるかを動的に学習するのではなく、見る必要のあるものだけを処理することができます。
+
+## {{% /tab %}}
```
Comment (alai97) [inline content/ja/security/cloud_workload_security/workload_security_rules.md:85]:
```suggestion
{{% /tab %}}
```

File: content/ja/security/cloud_workload_security/workload_security_rules.md [hunk 2]
```diff
@@ -134,7 +137,9 @@ Agent 式は、ホストやコンテナ内のアクティビティに基づい
 `default.policy` ファイルをターゲットホストの `{$DD_AGENT}/runtime-security.d` フォルダーにコピーします。このファイルには、ホスト上の `dd-agent` ユーザーの `read` と `write` のアクセス権が最低限与えられていることを確認してください。
 
 **注:** SCP や FTP などのユーティリティを使用する必要がある場合があります。
-{{% /tab %}}
+
+{{< /tabs >}}
```
Comment (alai97) [inline content/ja/security/cloud_workload_security/workload_security_rules.md:141]:
```suggestion
{{% /tab %}}
{{< /tabs >}}
```

**Review by alai97 — COMMENTED** [review]
These will need to be updated in Transifex as well.

File: content/ja/security/cloud_workload_security/workload_security_rules.md [hunk 3]
```diff
@@ -134,7 +137,10 @@ Agent 式は、ホストやコンテナ内のアクティビティに基づい
 `default.policy` ファイルをターゲットホストの `{$DD_AGENT}/runtime-security.d` フォルダーにコピーします。このファイルには、ホスト上の `dd-agent` ユーザーの `read` と `write` のアクセス権が最低限与えられていることを確認してください。
 
 **注:** SCP や FTP などのユーティリティを使用する必要がある場合があります。
+
 {{% /tab %}}
+{{< /tabs >}}
```
Comment (estherk15) [inline content/ja/security/cloud_workload_security/workload_security_rules.md:142]:
```suggestion

```

File: content/ja/security/cloud_workload_security/workload_security_rules.md (hunk 1 above)
Comment (estherk15) [inline content/ja/security/cloud_workload_security/workload_security_rules.md:85]:
```suggestion

```

**Review by estherk15 — APPROVED** [review]

**Pull request merged**